common/reviews/api/rush-lib.api.ts

```diff
@@ -156,6 +156,8 @@ class RushConfiguration {
   readonly eventHooks: EventHooks;
   findProjectByShorthandName(shorthandProjectName: string): RushConfigurationProject | undefined;
   findProjectByTempName(tempProjectName: string): RushConfigurationProject | undefined;
+  // (undocumented)
+  getCurrentProjectFromPath(currentFolderPath: string): RushConfigurationProject | undefined;
```

Comment:
Add docs.

Reply:
Why is this the "current" project?  Probably it should be `getProjectForPath`?

---
In reply to: [220408115](https://github.com/Microsoft/web-build-tools/pull/843#discussion_r220408115) [](ancestors = 220408115)